client/packages.go

```diff
@@ -90,13 +90,16 @@ type ResultInfo struct {
 // FindOptions supports exactly one of the following options:
 // - Refresh: only return snaps that are refreshable
 // - Private: return snaps that are private
-// - Query: only return snaps that match the query string
+// - Query: only return snaps that match the query string.
+// The SearchNameOnly predicate can be used to specify that
+// a search action is restricted to the snaps' name fields only. By default
+// the search is performed on all of the snaps' description fields.
 type FindOptions struct {
-	Refresh bool
-	Private bool
-	Prefix  bool
-	Query   string
-	Section string
+	Refresh        bool
+	Private        bool
+	SearchNameOnly bool
```

Comment:
I agree `Prefix` is a bad name for this. But I don't really like yours... but don't have a good suggestion. Let's see if somebody else has one!